api/Document.json

```
@@ -167,16 +167,16 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": 9
```

Versions need to be strings, like this:

```suggestion
              "version_added": "9"
```